/usr/local/lib/python3.8/dist-packages/torch/nn/modules/transformer.py:306: UserWarning: enable_nested_tensor is True, but self.use_nested_tensor is False because encoder_layer.self_attn.batch_first was not True(use batch_first for better inference performance)
  warnings.warn(f"enable_nested_tensor is True, but self.use_nested_tensor is False because {why_not_sparsity_fast_path}")
  0%|                                                                                                         | 0/99 [00:00<?, ?it/s]
[(226, 90, 370, 308, 'person'), (15, 377, 96, 479, 'container'), (102, 275, 170, 429, 'container'), (171, 273, 238, 409, 'container'), (340, 333, 381, 415, 'container'), (410, 361, 462, 447, 'soap_liquid'), (464, 367, 521, 458, 'container'), (434, 445, 474, 475, 'dent_floss'), (239, 377, 322, 453, 'tap')]
Traceback (most recent call last):
  File "../main.py", line 169, in <module>
    main()
  File "../main.py", line 154, in main
    train_model(config,train_ds,val_ds)
  File "/workspace/tfg_hhernandez/utils/train.py", line 102, in train_model
    output = model(features_noisy, frames)
  File "/usr/local/lib/python3.8/dist-packages/torch/nn/modules/module.py", line 1532, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/torch/nn/modules/module.py", line 1541, in _call_impl
    return forward_call(*args, **kwargs)
  File "/workspace/tfg_hhernandez/models/Model.py", line 34, in forward
    object_features=self.ObjFTExtractor(frames,max_objects=self.max_detected_objs)
  File "/usr/local/lib/python3.8/dist-packages/torch/nn/modules/module.py", line 1532, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/torch/nn/modules/module.py", line 1541, in _call_impl
    return forward_call(*args, **kwargs)
  File "/workspace/tfg_hhernandez/models/ObjectEmbeddingsGetter.py", line 72, in forward
    object_features = self.objects_in_one_frame(frames_list=frames, frame_num=middle_frame,max_objects=max_objects)
  File "/workspace/tfg_hhernandez/models/ObjectEmbeddingsGetter.py", line 302, in objects_in_one_frame
    text_ft = self.text_feature_extractor(key)
  File "/usr/local/lib/python3.8/dist-packages/torch/nn/modules/module.py", line 1532, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/torch/nn/modules/module.py", line 1541, in _call_impl
    return forward_call(*args, **kwargs)
  File "/workspace/tfg_hhernandez/models/TextEmbeddingsGetter.py", line 35, in forward
    raise ValueError(f"Key {key} not found in additional objects features")
ValueError: Key X1226_Y190_X2370_Y2308_CLSperson_F001632_VP_03 not found in additional objects features
